sljit_src/sljitNativeX86_common.c

```diff
@@ -496,35 +501,69 @@ static sljit_u32 execute_get_xcr0_low(void)
 {
 	sljit_u32 xcr0;
 
-#if defined(_MSC_VER) && _MSC_VER >= 1400
+#if (defined(_MSC_VER) && _MSC_VER >= 1400) || defined(__INTEL_COMPILER) \
+	|| (defined(__INTEL_LLVM_COMPILER) && defined(__XSAVE__))
```

Comment:
Does `immintrin.h` defines  `__XSAVE__` ?

Reply:
no,  calling the compiler with`-mxsave` or an equivalent does.

`immintrin.h` is needed for the definition of the intrinsic call

Reply:
Do we need `defined(__XSAVE__)` for the include above as well?

Reply:
I thought there was no harm on having it included regardless since the file is there anyway, but will do after testing.